### PR TITLE
 updated readme for cloud licensing and EA def

### PR DESCRIPTION
**Pull request by AjeyHiremath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,20 @@ For a detailed description of the driver, including a step by step deployment ex
...
 Prerequisite
 ------------
+
+A NIOS DDI Appliance with cloud automation License.
+
 To use the plugin, you need access to the Infoblox DDI product. For evaluation purposes, you can download a
 virtual version of the product from the [Infoblox Download Center](https://www.infoblox.com/infoblox-download-center).
-Alternatively, if you are an existing Infoblox customer, you can download it from the support site.
+Alternatively, if you are an existing Infoblox customer, you can download it from the support site and must be applied with
+cloud automation licensing from the CLI using ```set temp_license```.
```
Comment (jkraj) [inline README.md]: Remove this line           
 'and must be applied with 
+cloud automation licensing from the CLI using ```set temp_license```'

File: README.md [hunk 2]
```diff
@@ -11,15 +11,20 @@ For a detailed description of the driver, including a step by step deployment ex
 
 Prerequisite
 ------------
+
+A NIOS DDI Appliance with cloud automation License.
+
 To use the plugin, you need access to the Infoblox DDI product. For evaluation purposes, you can download a
 virtual version of the product from the [Infoblox Download Center](https://www.infoblox.com/infoblox-download-center).
```
Comment (jkraj) [inline README.md]: and you can also assign temp license by login into the Infoblox DDI appliance console with this command ```set temp_license```

File: README.md [hunk 3]
```diff
@@ -11,15 +11,20 @@ For a detailed description of the driver, including a step by step deployment ex
...
-Alternatively, if you are an existing Infoblox customer, you can download it from the support site.
+Alternatively, if you are an existing Infoblox customer, you can download it from the support site and must be applied with
+cloud automation licensing from the CLI using ```set temp_license```.
 
 Refer to [CONFIG.md](CONFIG.md) for details on vNIOS configuration.
 
-Build
------
-For dependencies and build instructions, refer to [BUILD.md](BUILD.md) .
+Development
```
Comment (jkraj) [inline README.md]: Move this section below 'Configuring Supported container runtimes' section

Comment (AjeyHiremath) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (AjeyHiremath) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (AjeyHiremath) [reply README.md]: added

Comment (AjeyHiremath) [reply README.md]: done

**Pull request merged**